content/riak/ts/1.6.0/using/querying/select/aggregate-functions.md

```diff
@@ -143,3 +146,29 @@ Returns `NULL` if no values were returned or all values were `NULL`.
 |-------------------|-------------|
 | sint64            | sint64 |
 | double            | double |
+
+### `PERCENTILE_DISC`, `PERCENTILE_CONT`
+
+Calculate a percentile, given as a value in the range [0..1], in a population of entries in the selection, with null values discarded.  The column argument must be of numeric type (`sint64`, `double` or `timestamp`).  Return type is the type of the column argument for `PERCENTILE_DISC`, and `double` for `PERCENTILE_CONT`.
```

Comment:
The DISC version always returns the value of the actual cell. It does so without casting it to some other type, and by that token, without double promotion.

The CONT version, on the other hand, does an interpolation step, which _does_ necessitate a double promotion. Which is why it is always 'double'.

(Or could it be that you are confused about the `x` and `0.3` in `PERCENTILE(x, 0.3)`? `x` is the argument column, `0.3` is the function positional parameter.)